Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -139,12 +211,73 @@ for temporary files of extensions and TYPO3 components.
 Directory for temporary files that should be public available
 (e.g. generated images).
 
-
 .. _directory-public-typo3temp-var:
 
-:file:`public/typo3temp/var/` or :file:`var/labels`
-"""""""""""""""""""""""""""""""""""""""""""""""""""
+:file:`public/typo3temp/var/`
+"""""""""""""""""""""""""""""
+
+Legacy installations only. Directory for temporary files that contains private
+files (e.g. cached Fluid templates) and should not be publicly available.
+See also :ref:`Environment-configuring-paths` for a more detailed description.
+
+.. _directory-var:
+
+:file:`var/`
+------------
 
-Directory for temporary files that contains private files (e.g. cached
-Fluid templates) and should not be publicly available.
+Composer installations only. Directory for temporary files that contains private files (e.g.
+cache and logs files) and should not be publicly available.
 See also :ref:`Environment-configuring-paths` for a more detailed description.
+
+.. _directory-var-cache:
+
+:file:`var/cache/`
+~~~~~~~~~~~~~~~~~~~
+
+Composer installations only. This directory contains internal files needed
+for the cache.
+
+.. _directory-var-labels:
+
+:file:`var/labels/`
+~~~~~~~~~~~~~~~~~~~
+
+Composer installations only.  The directory :file:`var/labels/` is for extension
+localisations. It contains all downloaded translation files.
+
+This path can be retrieved from the Environment API, see
+:ref:`Environment-labels-path`.
+
+.. _directory-var-log:
+
+:file:`var/log/`
+~~~~~~~~~~~~~~~~~~~
+
+Composer installations only.  This directory contains log files like the
+TYPO3 log, the deprecations log and logs generated by extensions.
+
+.. _directory-vendor:
+
+:file:`vendor/`
+---------------
+
+Composer installations only.  In this directory, which lies outside of
+the webroot, all extensions (system, third-party and custom) are installed
+as Composer packages.
+
+The directory contains folders for each required vendor and inside each
+vendor directory there is a folder with the different project names.
+
+For example the system extension `core` has the complete package name
+`typo3/cms-core` and will therefore be installed into the directory
+:file:`vendor/typo3/cms-core`. The extension `news`, package name
+`georgringer/news` will be installed into the folder
+:file:`vendor/georgringer/news`.
+
+Never put or symlink your extensions manually into this directory as they would
```

Comment:
Hmm, would rephrase it: "... as it is managed by Composer and any manual changes are getting lost, e.g. on deployment".